internal/tiltfile/docker.go

```diff
@@ -604,3 +609,15 @@ func (s *tiltfileState) dockerignoresForImage(image *dockerImage) []model.Docker
 
 	return s.dockerignoresForPaths(paths)
 }
+
+func (s *tiltfileState) checkForFastBuilds(manifests []model.Manifest) {
+Loop:
+	for _, m := range manifests {
+		for _, iTarg := range m.ImageTargets {
+			if !iTarg.AnyFastBuildInfo().Empty() {
+				s.warnings = append(s.warnings, fastBuildDeprecationWarning)
+				break Loop
```

Comment:
I find this style of breaking/continuing to labels very confusing. This could be rewritten as:

```golang
for _, m := range manifests {
                foundWarningForManifest := false
		for _, iTarg := range m.ImageTargets {
                        if foundWarningForManifest {
                                break
                        }
			if !iTarg.AnyFastBuildInfo().Empty() {
				s.warnings = append(s.warnings, fastBuildDeprecationWarning)
				foundWarningForManifest = true
                                break
```

This will work as it is, that's just my preference from previous label scar tissue (granted not in Go, though)

Reply:
or s/break Loop/return/?

Reply:
... 😅